packages/trader/src/Assets/Trading/Categories/Description/__tests__/vanilla-trade-description.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import VanillaTradeDescription from '../vanilla-trade-description';
+
+const mocked_props = {
+    onClick: jest.fn(),
+};
+
+describe('<VanillaTradeDescription />', () => {
+    it('a proper text of description should be rendered', () => {
+        render(<VanillaTradeDescription {...mocked_props} />);
+
+        expect(screen.getByText(/Vanilla options allow you to predict/i)).toBeInTheDocument();
+    });
+
+    it('specific text of description should be rendered if is_vanilla_fx is true', () => {
+        render(<VanillaTradeDescription {...mocked_props} is_vanilla_fx />);
+
+        expect(screen.getByText(/payout per pip/i)).toBeInTheDocument();
+        expect(screen.getByText(/You may sell the contract up to 24 hours before expiry/i)).toBeInTheDocument();
+    });
+
+    it('should call a function if word from vocabulary was clicked', () => {
+        render(<VanillaTradeDescription {...mocked_props} />);
+
+        const vocabulary_word = screen.getByText(/payout per point/i);
```

Comment:
@kate-deriv i know this place has been approved but I recommend using a `glossary_word` if this is what we mean here :) nit

Reply:
Sure, will do